benchmarks/cugraph/notebooks/feature_storage.ipynb

```diff
@@ -0,0 +1,171 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0e89dc09-ee78-4dae-8be0-3cf824e33079",
+   "metadata": {},
+   "source": [
+    "# FeatureStore Benchmarks"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "fdcc7186-cdbc-4673-91d1-211049e22072",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/datasets/vjawa/miniconda3/envs/all_cuda-115_arch-x86_64/lib/python3.9/site-packages/tqdm/auto.py:22: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
+      "  from .autonotebook import tqdm as notebook_tqdm\n"
+     ]
+    }
+   ],
+   "source": [
+    "import cudf\n",
+    "import cupy as cp\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "\n",
+    "# Import FeatureStore class\n",
+    "from cugraph.gnn import FeatureStore"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfcf3cc5-dd77-4a88-810e-0bcd2c7becd4",
+   "metadata": {},
+   "source": [
+    "# Feature to Load"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "439f1de5-16fa-432a-9a17-7b00e1d6f63b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "numpy_path = '/raid/gnn/'\n",
+    "paper_feat_path = numpy_path + 'paper/node_feat.npy'\n",
+    "paper_ar = np.load(paper_feat_path, mmap_mode='r') "
```

Comment:
Will these paths work for everyone?

Reply:
I can change these to ASE-lab's `/datasets` mount if that helps . The dataset is 185gb so i cant really save disk.  Does that sound reasonable ?

Reply:
Sure, whatever you think will be reasonable. I was mostly just raising awareness since others might want to run them soon.

Reply:
Resolved by https://github.com/rapidsai/cugraph/pull/3139/commits/ae689fcd6609e3dcd946b7240c9e75ce6f5d7a1a